org.eclipse.corrosion.tests/src/org/eclipse/corrosion/tests/TestLSPIntegration.java

```diff
@@ -82,6 +82,7 @@ protected boolean condition() {
 		}.waitForCondition(editor.getEditorSite().getShell().getDisplay(), 30000);
 		IMarker marker = file.findMarkers(IMarker.PROBLEM, true, IResource.DEPTH_ZERO)[0];
 		assertTrue(marker.getType().contains("lsp4e"));
-		assertEquals(3, marker.getAttribute(IMarker.LINE_NUMBER, -1));
+		//info marker on first line
+		assertEquals(1, marker.getAttribute(IMarker.LINE_NUMBER, -1));
```

Comment:
I wouldn't merge it before a more careful audit of why that changed.

Reply:
There is info marker shown on the first line having the same text as the error marker on the third one. I guess this is coming from RLS or LSP4E.

Reply:
I see it when running Corrosion locally.

Reply:
So there are now 2 markers instead of one? In that case, I'd blame LSP4E.
Please update the test so that it at least checks that a marker is visible on the expected line (ignoring the one on line 1), and we could merge it.

Reply:
Latest patch checks that there is the marker on line 3 by iterating through all markers